exporter/importables.go

```diff
@@ -2294,4 +2294,40 @@ var resourcesMap map[string]importable = map[string]importable{
 		},
 		// TODO: add Depends & Import to emit corresponding UC Volumes when support for them is added
 	},
+	"databricks_storage_credential": {
+		WorkspaceLevel: true,
+		Service:        "storage_credentials",
```

Comment:
let's make it explicit that it's uc-related and use something like `uc-storage-credentials`, like here: https://registry.terraform.io/providers/databricks/databricks/latest/docs/guides/experimental-exporter#uc-artifact-allowlist